### DISCUSSION TITLE
Error on composer update but only in one server: Class "App\Filament\Resources\OldDbEventResource\Pages\ListOldDbEvents" not found

### DISCUSSION THREAD

**Discussion by giacomomasseron** [filamentphp/filament#11577] [Help]
### Package

Panel builder

### Package Version

v3.2.37

### How can we help you?

Hi,  
I get this error when I execute the command composer dump-autoload (or any other command).

The error is:

```
  Class "App\Filament\Resources\OldDbEventResource\Pages\ListOldDbEvents" not found

  at app/Filament/Resources/OldDbEventResource.php:129
    125▕
    126▕     public static function getPages(): array
    127▕     {
    128▕         return [
  ➜ 129▕             'index' => ListOldDbEvents::route('/'),
    130▕             'create' => CreateOldDbEvent::route('/create'),
    131▕             'edit' => EditOldDbEvent::route('/{record}/edit'),
    132▕         ];
    133▕     }

```
The strange things are:

- in local, same exact code, it works
- in another server, it works
- I checked the existance of the file, and it does exist

I suppose the problem is somewhere else, but I am not able to figure out.

**Comment by leandrocfe** [comment]
Did you try to import the `ListOldDbEvents` class again?
Maybe a case sensitive issue in the names?

**Comment by giacomomasseron** [comment]
I did it, the problem was that our deployer was not uploading some files.

**Reply by leandrocfe** [reply]
*I checked the existance of the file, and it does exist* 🤷‍♂️

**Reply by giacomomasseron** [reply]
You right, I has been dumb :(
